exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServerTest.java

```diff
@@ -117,12 +117,6 @@ void fetchPrometheus(String endpoint) {
             "# TYPE target info\n"
                 + "# HELP target Target metadata\n"
                 + "target_info{kr=\"vr\"} 1\n"
-                + "# TYPE otel_scope_info info\n"
-                + "# HELP otel_scope_info Scope metadata\n"
-                + "otel_scope_info{otel_scope_name=\"grpc\",otel_scope_version=\"version\"} 1\n"
-                + "# TYPE otel_scope_info info\n"
-                + "# HELP otel_scope_info Scope metadata\n"
-                + "otel_scope_info{otel_scope_name=\"http\",otel_scope_version=\"version\"} 1\n"
                 + "# TYPE grpc_name_total counter\n"
                 + "# HELP grpc_name_total long_description\n"
                 + "grpc_name_total{otel_scope_name=\"grpc\",otel_scope_version=\"version\",kp=\"vp\"} 5.0 0\n"
```

Comment:
@dashpole In this example we drop the otel_scop_info info metric because the scope attributes are empty, but we continue to include the `otel_scope_name` and `otel_scope_version` attributes on the metrics part of the scope. 

This seems to be aligned with the spirit of https://github.com/open-telemetry/opentelemetry-specification/pull/3660, but its a bit of a gray area.

Reply:
That is the intended behavior.  It allows reconstructing the scope without the additional info metric.